### DISCUSSION TITLE
Q&A

### DISCUSSION THREAD

**Discussion by abdelghafour77** [abdelghafour77/contact-and-landing-page-v2#1] [Q&A] [answered]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by anas1k** [comment, marked as answer]
yoo

**Reply by abdelghafour77** [reply]
yooo
